verilog/tools/ls/symbol-table-handler.cc

```diff
@@ -317,6 +342,59 @@ std::vector<verible::lsp::Location> SymbolTableHandler::FindReferencesLocations(
   return locations;
 }
 
+verible::lsp::Range SymbolTableHandler::FindRenameLocations(
```

Comment:
What comes back is not plural, but returns a single range. And it is optional.
Maybe `RenameableRangeAtCursor()` or something like that ?

Reply:
As with the YAML spec, it is a leftover from when it did return a `Range[]`. Changed it